lib/util/backend_data.dart

```diff
@@ -0,0 +1,70 @@
+/// enumeration for gender-variable.
+enum Gender {
+  /// element for denoting the title of men
+  male,
+
+  /// element for denoting the title of women
+  female,
+}
+
```

Comment:
Please include at least "other"

Reply:
Another option would be to name the enum "Sex", because other than genders there can only be two or three sexes. Also I think that the biological constitution is in a medical application more relevant rather than what the person identifies with.

Reply:
Here, I added the option "diverse".

Reply:
Finally, I have decided to leave it as a binary option as shown by the mockup. We should have a discussion with the other Stakeholders.